hca/dss/__init__.py

```diff
@@ -504,6 +534,15 @@ def _download_file(self, dss_file, dest_path, num_retries=10, min_delay_seconds=
                 raise ValueError("Expected sha256 {} Received sha256 {}".format(
                     dss_file.sha256.lower(), download_hash.lower()))
 
+    def _make_path_if_necessary(self, dest_path):
+        directory, _ = os.path.split(dest_path)
+        if directory:
+            try:
+                os.makedirs(directory)
```

Comment:
Instead of using the try-catch. Do...
```suggestion
            os.makedirs(directory, exist_ok=True)
```

Reply:
Nevermind the `exist_ok` keyword arg in `os.makedirs()` doesn't exist in python2